src/perfcollect/perfcollect

```diff
@@ -1373,7 +1414,6 @@ ProcessCollectedData()
     local directoryName=$traceName$traceSuffix
     mkdir $directoryName
 
-    # Save LTTng trace files.
```

Comment:
I think this deletion was unintentional?

Reply:
Yes I'll revert it